framework/lib/components/tags-input/tags-input.tsx

```diff
@@ -125,7 +127,7 @@ export function TagsInput<T extends Option> ({
 
   const handleFocus = () => {
     setIsFocused(true)
-    if (options && options.length > 0) {
+    if (options && options?.length > 0) {
```

Comment:
`if(options?.length) {...}`

Reply:
This throws an error that options?.length might be undefined

Reply:
Weird. Are you trying to compare it to 0? Just do the direct value:
Undefined and 0 is false.